stdlib/public/SDK/Foundation/NSStringAPI.swift

```diff
@@ -1023,20 +1027,19 @@ extension String {
     scheme tagScheme: String,
     options opts: NSLinguisticTaggerOptions = [],
     orthography: NSOrthography? = nil,
-    tokenRanges: UnsafeMutablePointer<[Range<Index>]> = nil // FIXME:Can this be nil?
+    tokenRanges: UnsafeMutablePointer<[Range<Index>]>? = nil // FIXME:Can this be nil?
   ) -> [String] {
     var nsTokenRanges: NSArray? = nil
-    let result = tokenRanges._withBridgeObject(&nsTokenRanges) {
+    let op = {
```

Comment:
I spent a good fifteen minutes trying to figure out what it did, and managed to replace all the similar APIs. I really think just expanding it out inline makes the code easier to read.